src/VisualStudio/Core/Def/Implementation/Watson/WatsonReporter.cs

```diff
@@ -31,19 +31,19 @@ internal static class WatsonReporter
         /// Report Non-Fatal Watson
         /// </summary>
         /// <param name="exception">Exception that triggered this non-fatal error</param>
-        public static void Report(Exception exception)
+        public static void Report(Exception exception, bool critical = false)
```

Comment:
>bool critical = false [](start = 55, length = 21)

maybe enum is better

Reply:
@ivanbasov  do you want enum with every thing? or just Critical and None?

Reply:
I think that we may want the enum on our side to differ from the enum on the Telemetry side. So, we should define the enum for our purposes. Currently, it should contain Critical and None only.

Reply:
alright. will do!

Reply:
@ivanbasov  done!